client/profile-wizard/index.js

```diff
@@ -215,6 +221,17 @@ class ProfileWizard extends Component {
 		}
 	}
 
+	async skipProfiler() {
+		const { updateProfileItems } = this.props;
+		await updateProfileItems( {
```

Comment:
If `updateProfileItems` errors, how is it handled?

Reply:
I think error handling might be good to handle before merging since this could end in being redirected back here if updating fails ( failure -> navigate to dashboard -> redirect back to profiler since not skipped ).

Maybe at least we could move the below `recordEvent` and `geHistory` calls inside `updateProfileItems().then( // ... here );` so they only fire on success.